CMakeLists.txt

```diff
@@ -403,7 +406,7 @@ if(ENABLE_CPACK)
           "${CPACK_RPM_PACKAGE_REQUIRES},qt5-qtbase,qscintilla-qt5")
     endif()
 
-    if(PYTHON_VERSION_MAJOR EQUAL 3)
+    if(Python_VERSION_MAJOR EQUAL 3)
```

Comment:
Now that we are no longer supporting python 2 do we need these python major version checks?

Reply:
You're right. We don't need this anymore.